### PR TITLE
chore: AvatarHUD as legacy

### PR DESCRIPTION
**Pull request by pbosio**
set current AvatarHUD as legacy to start working in the new version

### CODE REVIEW

**Review by sandrade-dcl — APPROVED** [review]
I understand that this change is only for

**Review by pravusjif — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDController.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
+namespace Legacy
+{
+    public class AvatarHUDController : IHUD
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDController.cs:3]: Maybe we can take advantage of this opportunity to change the name to something less confusing, like `ProfileHud`, `IdentityToastHud` or whatever (not sure if this is in the scope of this PR, just general feedback).

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDView.cs [hunk 2]
```diff
@@ -0,0 +1,105 @@
+﻿using System.Runtime.CompilerServices;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using DCL.Interface;
+using Image = UnityEngine.UI.Image;
+
+[assembly: InternalsVisibleTo("AvatarHUDTests")]
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDView.cs:8]: We should move this to `AssemblyInfo.cs` (again, general feedback).

**Review by BrianAmadori — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDController.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDController.cs:3]: You are right, I will use the suggested names for the new HUD

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDView.cs (hunk 2 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarHUD/Legacy/AvatarHUDView.cs:8]: All that code will end in the trash after the new HUDs go live, so I don't really see the point of creating a new file to delete it later.
but I will do what you said for the new implementation

**Pull request merged**